ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml

```diff
@@ -255,3 +255,17 @@ SENSOR_ZTM_ZHM_CO2M_VOCM_PMM:
   - CO2M
   - VOCM
   - PMM
+  opt_uses:
```

Comment:
It seems you are adding enough fields to warrant a new canonical type. There is distinct functionality (e.g., the sound level monitoring) that should not be added to this type (which is focused around air quality)